resolwe_bio/descriptors/sample_vaccinesurvey.yml

```diff
@@ -0,0 +1,134 @@
+- slug: sample-vaccinesurvey
+  name: Vaccinesurvey sample annotation
+  description: Vaccinesurvey sample annotation
+  schema:
+    - name: sample
+      label: Sample annotation
+      group:
+        - name: annotator
+          label: Annotator/Scientist
+          type: basic:string
+          description: >
+            The name of the person that prepared the data.
+        - name: study_code
+          label: Study code identifying patient
+          type: basic:string
+        - name: sex
```

Comment:
I just tried to be politically correct, since there are so many types of gender these days :-)